<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myfonts_folders</class>
 <widget class="QWidget" name="myfonts_folders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget  {
	background-color: #191d20;
}

* {
	font: 57 12pt &quot;Roboto Medium&quot;;
}

QPushButton#addNewFolderIcon { 
	image: url(:/assets/icons/addNewFolder.svg);
}
QPushButton#addNewFolderIcon:hover { 
	image: url(:/assets/icons/addNewFolderHOVER.svg);
}

/* SLIDER */


/*  SCROLLBAR */

QListView:focus {
   border: none;
   outline: none;
}

QScrollBar:vertical {
     background: #191d20;
     width: 15px;
     margin: 22px 0 22px 0;
 }
 QScrollBar::handle:vertical {
     background: rgb(74, 85, 95);
     min-height: 20px;
 }

 QScrollBar::handle:vertical:hover {
     background:rgb(99, 114, 127)
 }

 QScrollBar::add-line:vertical {
     background: #191d20;
     height: 20px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     background: #191d20;
     height: 20px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: none;
     width: 3px;
     height: 3px;
     background: #37474f;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: #191d20;
 }</string>
  </property>
  <widget class="QListWidget" name="folderList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>560</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="spacing">
    <number>40</number>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="addNewFolder_const" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <widget class="QPushButton" name="addNewFolderIcon">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>75</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="favFolder_const" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 52, 58);
color: #d9d9d9;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Favourites&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
